Source/WebCore/svg/animation/SMILTimeContainer.cpp

```diff
@@ -247,7 +247,10 @@ void SMILTimeContainer::sortByPriority(AnimationsVector& animations, SMILTime el
 {
     if (m_documentOrderIndexesDirty)
         updateDocumentOrderIndexes();
-    std::ranges::sort(animations, PriorityCompare(elapsed));
+
+    // Only sort if not already sorted.
+    if (!std::ranges::is_sorted(animations, PriorityCompare(elapsed))) [[unlikely]]
```

Comment:
Why is this `[[unlikely]]`?

The `SVGSMILElements` need to be sorted based on their priorities. Their priorities depend on their begin times relative to the elapsed time. So every time `SMILTimeContainer::updateAnimations()` is called, the `AnimationsVectors` are expected to be sorted differently.